src/translation/translation_de_DE.ts

```diff
@@ -4805,32 +4805,32 @@ Wir haben Deinen Kanal stummgeschaltet und die Funktion &apos;Stummschalten&apos
     <message>
         <location filename="../util.cpp" line="1733"/>
         <source>This app uses the following libraries, resources or code snippets:</source>
-        <translation type="unfinished">Diese Applikation verwendet die folgenden Bibliotheken, Ressourcen oder Codeschnipsel:</translation>
+        <translation>Diese App verwendet die folgenden Bibliotheken, Ressourcen oder Codeschnipsel:</translation>
     </message>
     <message>
         <location filename="../util.cpp" line="1735"/>
         <source>Qt cross-platform application framework</source>
-        <translation type="unfinished">Qt plattformübergreifender Anwendungsrahmen</translation>
+        <translation>Qt plattformübergreifendes Framework</translation>
```

Comment:
Do we check this string against the translations that Qt uses for their framework?

Reply:
No. 
But checking Wikipedia https://de.m.wikipedia.org/wiki/Qt_(Bibliothek)
Maybe it's ok?